content/open-source/sigstore/cosign/an-introduction-to-cosign.md

```diff
@@ -6,6 +6,7 @@ description: "Understanding Cosign, a project under Sigstore"
 date: 2022-19-07T08:49:31+00:00
 lastmod: 2023-03-11T08:49:31+00:00
 draft: false
+tags: ["cosign", "Overview"]
```

Comment:
If "Rekor" is title case, I would proabbly make "Cosign" title case too. "policy-controller" is maybe the exception here